### ISSUE TITLE
verify-flags-underscore.py: Found illegal 'flag' usage in cluster/mesos/docker/common/bin/util-ssl.sh

### ISSUE THREAD

**Issue by ixdy** [kubernetes/kubernetes#12998]
In a recent Shippable run, I noticed that hack/verify-flags-underscore.py is reporting an apparent error, though it's not failing the build:

```
./hack/verify-flags-underscore.py
Found illegal 'flag' usage. If these are false positives you should running `hack/verify-flags-underscore.py -e > hack/verify-flags/exceptions.txt` to update the list.
cluster/mesos/docker/common/bin/util-ssl.sh: local cluster_domain="cluster.local"
```

So two questions:
- should this "illegal flag usage" have failed the build?
- should the named mesos script fix its flags? or should this be whitelisted?

@eparis @karlkfi

**Comment by eparis** [comment]
should be white listed.  should have failed.  I figure out why it didn't fail...

**Comment by eparis** [comment]
both things fixed in #13020
